test/extended/setup.sh

```diff
@@ -159,7 +155,7 @@ function os::test::extended::run () {
 		return
 	fi
 
-	"${GINKGO}" -v "${runArgs[@]}" "${EXTENDEDTEST}" "$@"
+	ginkgo -v "${runArgs[@]}" extended.test "$@"
```

Comment:
Yes, you're right -- we should be passing the path always when we're not invoking it directly.